### PR TITLE
book list

### PR DESCRIPTION
**Pull request by jonnyalexbh**
## Summary
- book list

## Trello Card
https://trello.com/c/K4gBYLLq/30-listado-de-libros

## Screenshot
Screen Shot 2019-12-05 at 12 23 25 PM

### CODE REVIEW

File: book-list.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training | book list</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+<body>
+  <div class="card-list">
+    <div class="card-content">
```
Comment (betoobernal) [inline book-list.html]: tag needless

File: scss/list.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  box-shadow: 6px 6px 5px rgba($black, 0.4);
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  margin: 0 35px 20px 35px;
+  padding: 26px 0 0 28px;
+  width: 198px;
+}
+
+.card img {
```
Comment (betoobernal) [inline scss/list.scss]: don't use a tag name as a selector

Comment (jonnyalexbh) [reply scss/list.scss]: ready 👍

File: book-list.html (hunk 1 above)
Comment (jonnyalexbh) [reply book-list.html]: ready 💯

**Review by betoobernal — APPROVED** [review]

**Pull request merged**